packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/ManagedACTAccessRequirementFormWikiWrapper.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { useGetAccessRequirementWikiPageKey } from '../../../synapse-queries'
+import { useCanShowManagedACTWikiInWizard } from '../AccessRequirementListUtils'
+import { Grid, Typography } from '@mui/material'
+import MarkdownSynapse from '../../Markdown/MarkdownSynapse'
+
+type ManagedACTAccessRequirementFormWikiWrapperProps = React.PropsWithChildren<{
+  managedACTAccessRequirementId: string
+}>
+
+/**
+ * Renders the child content next to the wiki associated with the provided ManagedACTAccessRequirement. Utilizes a grid
+ * layout where the wiki content will not appear on small screen sizes.
+ *
+ * @param props
+ * @constructor
+ */
+export default function ManagedACTAccessRequirementFormWikiWrapper(
+  props: ManagedACTAccessRequirementFormWikiWrapperProps,
+) {
+  const { children, managedACTAccessRequirementId } = props
+  const { data: wikiPage } = useGetAccessRequirementWikiPageKey(
+    managedACTAccessRequirementId,
+    {
+      enabled: !!managedACTAccessRequirementId,
+    },
+  )
+
+  const canShowWiki = useCanShowManagedACTWikiInWizard()
+
+  return (
+    <Grid container spacing={5}>
+      <Grid
+        item
+        xs={12}
+        md={6}
+        lg={5}
+        sx={{
+          maxHeight: '500px',
+          overflow: 'auto',
+          pr: {
+            xs: 0,
+            md: 1,
+          },
```

Comment:
Conditionally add right padding because the content may be too close to the scrollbar if the wiki is shown